x-pack/plugins/watcher/public/directives/moment.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment';
+import _ from 'lodash';
+import { uiModules } from 'ui/modules';
+
+uiModules
+  .get('xpack/watcher')
+  .filter('moment', function (config) {
+    return function (datetime) {
+      const format = config.get('dateFormat');
```

Comment:
Can be simplified to:

```js
return moment(datetime).format(config.get('dateFormat'));
```

Reply:
The code in watcher is being refactored by @bmcconaghy ATM
So I'm not changing anything there (just moved the filter to the watcher folder)